tests/acceptance/features/user/super_user/add.feature

```diff
@@ -7,5 +7,5 @@ Feature: Add new User
   Scenario: 1016-1245 Adding new user
     Given I log in as a user with role "Super User"
     Then I add a new user with "Super User" role
-
-    Then I delete the User
\ No newline at end of file
+    Then I verify an email was sent
```

Comment:
This should probably be a second test, so that creating a user can be tested separately from the email being sent (such as if the email sending is disabled).

Reply:
Moved the test into it's own email area in tests/acceptance/features/emails/super_user/user_create.feature